dist/tools/ci/print_toolchain_versions.sh

```diff
@@ -8,6 +8,10 @@ get_cmd_version() {
     local cmd="$1"
     if command -v "$cmd" 2>&1 >/dev/null; then
         ver=$("$cmd" --version 2> /dev/null | head -n 1)
+        # some tools (eg. openocd) print version info to stderr
```

Comment:
Why not just get both output directly ?

Silencing stderr was done in your previous PR https://github.com/RIOT-OS/RIOT/commit/9968889c1302be783fd7cfffa9106279c7fee291 but the first version of the PR did not do it according to the code shown in https://github.com/RIOT-OS/RIOT/pull/8727#discussion_r171792583 and nothing explained why.

Also none of the previous code was redirecting `stderr` and it was introduced without justification in any commit or comment. Do you know why you added it ?

Reply:
I meant to put it on the line with the second `ver` calculation but somehow miss-clicked.